x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java

```diff
@@ -520,6 +521,19 @@ private void checkAutoFollowPattern(String autoFollowPattenName,
                             }
                             groupedListener.onResponse(new Tuple<>(indexToFollow, failure));
                         });
+                    } else if (SearchableSnapshotsConstants.isSearchableSnapshotStore(leaderIndexSettings)) {
+                        String message = String.format(Locale.ROOT,
+                            "index to follow [%s] is a searchable snapshot index and cannot be used for cross-cluster replication purpose",
+                            indexToFollow.getName()
+                        );
+                        LOGGER.warn(message);
```

Comment:
> I wonder if this needs to be a warning? It seems like a documented behavior and therefore this could be just DEBUG?

I did not find any documentation relative to this - hence the initial WARN level - but maybe you found something in the CCR doc? I pushed e8ab12a5a91c954016caa43a3e43cef0c426883c to move to debug.

> I wonder if we should update docs for auto-follow to state that searchable snapshots cannot be followed and will be ignored by auto-follow patterns?

I think we should and I'd like to do a follow up PR for this.

Reply:
I did not see any such info, I just thought it was more logical to simply ignore these than to warn, in particular because of the issues to use `*` patterns when auto-following like #67686

Reply:
Thanks, I agree.